GetandTake/Pages/Shared/_Header.cshtml

```diff
@@ -1,45 +1,49 @@
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link" href="#">About</a>
-                        </li>
-                        <li class="nav-item dropdown">
-                            <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Dropdown</a>
-                            <div class="dropdown-menu">
-                                <a class="dropdown-item" href="#">Action</a>
-                                <a class="dropdown-item" href="#">Another action</a>
-                                <a class="dropdown-item" href="#">Something else here</a>
-                                <div class="dropdown-divider"></div>
-                                <a class="dropdown-item" href="#">Separated link</a>
-                            </div>
-                        </li>
-                    </ul>
-                    <form class="d-flex">
-                        <input class="form-control me-sm-2" type="text" placeholder="Search">
-                        <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
+﻿<div class="container mb-4">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand" asp-page="/Index">Get & Take</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarColor01">
+                <ul class="navbar-nav me-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-page="/Categories/Category">Categories<i class="bi bi-diagram-2"></i></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-page="/Products/Product">Products<i class="bi bi-file-plus-fill"></i></a>
+                    </li>
+                    @*                        <li class="nav-item">
+                    @if (!User.Identity.IsAuthenticated)
+                    {
+                    <a class="nav-link" asp-area="" asp-page="/Account/Login">Login<i class="bi bi-file-plus-fill"></i></a>
+                    }
+                    else
+                    {
+                    <form method="post" asp-page="/Account/Login" asp-page-handler="Logout">
+                    <div class="input-group">
+                    <span>@User.Identity.IsAuthenticated</span>
+                    <button type="submit" class="btn btn-link">Logout</button>
+                    </div>
                     </form>
-                </div>
+                    }
+                    </li>*@
```

Comment:
Try to **always** remove commented code, it is a bad practice to leave it commented (in 90% of cases you will not get back to it), in other cases you should stick with YAGNI rule